code/modules/surgery/eye_cybernetic_implant.dm

```diff
@@ -0,0 +1,31 @@
+/datum/surgery/eye_cybernetic_implant
+	name = "eye cybernetic implant"
+	steps = list(/datum/surgery_step/incise, /datum/surgery_step/clamp_bleeders, /datum/surgery_step/retract_skin, /datum/surgery_step/cybernetic_implant, /datum/surgery_step/fix_eyes, /datum/surgery_step/close)
+	location = "eyes"
+
+/datum/surgery_step/cybernetic_implant
+	accept_any_item = 1
```

Comment:
To differentiate what kind of implant you're going to implant, as eye implants are the not the only ones. This was my first contact with surgery though and I will go over this code again.